### DISCUSSION TITLE
EcmaScript version to target

### DISCUSSION THREAD

**Discussion by petarvujovic98** [near/near-sdk-js#207] [Ideas]
Are there any restrictions on the ES version we should target when building the contracts?
If not can we allow the users to choose their ES version via the `tsconfig.json` file.

If there are, or we cannot let users choose the ES version on their own, I think we should use the newest version by default (e.g. currently the newest version is ES2022/ES13)

**Comment by volovyks** [comment]
Yes, the version that is specified in the projects project.json is not affecting the build process. It can be confusing.
I would just KISS it and use the latest version for now.
